analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -297,3 +330,25 @@ internal fun Identifier.toVcsInfo(): VcsInfo {
     val vcsInfo = VcsHost.toVcsInfo("https://$name")
     return vcsInfo.copy(revision = getRevision(version))
 }
+
+/**
+ * Parse the given [lines] generated by the _go mod why_ command and the names of packages that are reported to be
+ * used by the main module.
+ * See https://golang.org/ref/mod#go-mod-why.
+ */
+internal fun parseWhyOutput(lines: Sequence<String>): Set<String> {
```

Comment:
How about making the parameter just `String` for simplicity?

Reply:
You mean the call to `lineSequence()` should be done in the function?

Reply:
Yes.